crates/ide/src/annotations.rs

```diff
@@ -188,21 +203,23 @@ mod tests {
 
     use crate::{fixture, Annotation, AnnotationConfig};
 
-    fn check(ra_fixture: &str, expect: Expect) {
+    use super::AnnotationLocation;
+
+    const DEFAULT_CONFIG: AnnotationConfig = AnnotationConfig {
+        binary_target: true,
+        annotate_runnables: true,
+        annotate_impls: true,
+        annotate_references: true,
+        annotate_method_references: true,
+        annotate_enum_variant_references: true,
+        annotation_location: AnnotationLocation::AboveName,
+    };
+
+    fn check(ra_fixture: &str, expect: Expect, config: &AnnotationConfig) {
```

Comment:
It might make sense to introduce a `check_with_config` function and have `check` call that with the `DEFAULT_CONFIG` instead

Reply:
Works for me!